locale/circuitpython.pot

```diff
@@ -3019,7 +3028,7 @@ msgstr ""
 msgid "complex values not supported"
 msgstr ""
 
-#: extmod/moduzlib.c shared-module/zlib/DecompIO.c
+#: extmod/moduzlib.c
```

Comment:
This change wasn't intentional. I'm not sure why it's here. I'll check main branch and match that.

Reply:
When I run `make translate` the `shared-module/zlib/DecompIO.c` portion of this line gets removed. I noticed that this portion is present in main branch. I copied it back into this branch but after running `make translate` it gets removed again.

Anyone know what could cause that to be happening in my local repo? It looks like the file that it references is not present in main, so maybe it's intended for this portion to be removed from the .pot file? 

For now I will put it back how it is in main because I don't fully understand the ramifications of it existing vs. not here. Ideally I'd like to figure out what I need to do to my local instance to make it the same as main.

Reply:
I'd suggest that you merge or rebase from upstream. I think your local repo is out of date with respect to what messages are needed, so when you run `make translate`, you don't get the latest. Note that the line in question is just a comment too.